src/components/website-management/website-form-components/customization/customization-components/CusPopUp.js

```diff
@@ -0,0 +1,98 @@
+import axios from "axios";
+
+
+const CusPopUp = (props) => {
+
+    const { openPopup, handleClose,data, getTemps} =props;
+
+    const [textBox01,setTextBox01] =useState();
+
+    const [textBox02,setTextBox02] =useState();
+
+    const [textBox03,setTextBox03] =useState();
+
+    const template = {
+        templateName:textBox01,
+        templateRegulation:textBox02,
+        templateContent:textBox03
+    }
+
+    const handleSave =async () =>{
+        /*console.log(data);
+        console.log(textBox01);
+        console.log(textBox02);
+        console.log(textBox03);*/
+
+        const id =data.templateId;
+        await axios.put(`http://localhost:8080/api/template/updateTemplateID/${id}`, template).then(() => getTemps());
+
+        handleClose();
+        console.log(data);
+        console.log(textBox01);
+        console.log(textBox02);
+        console.log(textBox03);
+    }
+
+    useEffect(() =>{
+        console.log("data",data);
+
+         if(data){
+            setTextBox01(data.templateName);
+            setTextBox02(data.templateRegulation);
+            setTextBox03(data.templateContent);
+        }
+
+    },[data]);
+
+    return (
+        <div className={"dialog-box"}>
+            <Dialog open={openPopup} maxWidth={"lg"}>
+                <DialogTitle>
+                    <div className={"dialogTitle-container"} style={{flexDirection:"row",
+                        backgroundColor:"#ccfefd",
+                        padding:5,
+                        paddingTop:8,
+                        height:'50px',
+                        display:"flex",
+                        justifyContent:"space-between",
+                        borderRadius:5
+                    }}>
+                        <div style={{float:"left",}}><Typography variant={"h6"} sx={{color:"#004587",fontWeight:"bold"}}>Cookie Template</Typography></div>
+                        <div style={{float:"right",}}><Button onClick={handleClose} startIcon={<CancelIcon/>} sx={{color:"red"}}> </Button></div>
+                    </div>
+                </DialogTitle>
+                <DialogContent>
+                        <div className={"dialogContent-container"} style={{flexDirection:"row",width:"800px",justifyContent:"space-between"}}>
+                            <div style={{paddingTop:10,paddingBottom:10}}><TextField sx={{width:'50ch'}} id={data.templateId} label="Template Name" variant="outlined" value={textBox01} onChange={(e) =>setTextBox01(e.target.value) }/></div>
+                            <div style={{paddingTop:10,paddingBottom:10}}><TextField sx={{width:'50ch'}} id={data.templateId} label="Privacy Regulation Type" value={textBox02} onChange={(e) =>setTextBox02(e.target.value)} /></div>
+                            <div style={{paddingTop:10,paddingBottom:10}}><TextField sx={{width:'50ch'}} id={data.templateId} label="Content" variant="outlined" value={textBox03} onChange={(e) =>setTextBox03(e.target.value)} multiline={true} /></div>
```

Comment:
Format the JSX. Use a tool such as prettier to automate it.

Reply:
fixed